integration_tests/test_cmake_plugin.py

```diff
@@ -1,6 +1,6 @@
 # -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
 #
-# Copyright (C) 2015 Canonical Ltd
+# Copyright (C) 2015, 2017 Canonical Ltd
```

Comment:
well, in this one you can keep as is, but I am certain there were changes in 2016 here